public/app/plugins/panel/state-timeline/hooks.tsx

```diff
@@ -45,8 +36,6 @@ export function usePagination(frames?: DataFrame[], perPage?: number) {
     };
   }
 
-  perPage ||= defaultOptions.perPage!;
```

Comment:
realized as I was writing the last few tests that this case is impossible to reach, because we check for `!perPage` just above.